addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/ParamMinerConfig.java

```diff
@@ -60,14 +60,17 @@ public class ParamMinerConfig {
     private String urlJsonIncludeString;
     private boolean urlPostRequest;
     private boolean urlGetRequest;
+    private String customHeaderWordlistPath;
+    private String customCookieWordlistPath;
+    private int urlGuessChunkSize;
 
     public ParamMinerConfig() {
         this.url = "";
         this.addFcbzCacheBuster = false;
         /** This would allow user to use separate wordlists. */
-        this.usePredefinedUrlWordlists = true;
-        this.usePredefinedHeaderWordlists = true;
-        this.usePredefinedCookieWordlists = true;
+        this.usePredefinedUrlWordlists = false;
+        this.usePredefinedHeaderWordlists = false;
+        this.usePredefinedCookieWordlists = false;
```

Comment:
This could be removed now, they are already false by default (same for the others).